settings.ts

```diff
@@ -135,5 +137,15 @@ export class AutoLinkTitleSettingTab extends PluginSettingTab {
             await this.plugin.saveSettings();
           })
       );
+
+    new Setting(containerEl)
+      .setName("LinkPreview API Key")
+      .setDesc("API key for the LinkPreview.net service. Get one at https://my.linkpreview.net/access_keys")
+      .addText(text => text
+        .setValue(this.plugin.settings.linkPreviewApiKey || "")
+        .onChange(async (value) => {
+          this.plugin.settings.linkPreviewApiKey = value;
```

Comment:
Also, lets `value.trim()` to remove any erroneous whitespace from the user copy/pasting their API key.